### PR TITLE
fix the style of hero call to action button

### PR DESCRIPTION
**Pull request by jibola2588**
## Description 📝
fix issue number #184 
The font size of the call to action button was adjusted, presumably to improve readability or visual consistency.

The button's position was altered, likely to ensure it is aligned centrally within its container div or to enhance its visual appeal.

 A hover effect was implemented on the button, which typically means that when a user hovers their mouse cursor over the button, it exhibits some form of visual change or animation to provide feedback and make it more interactive.

## Changes 💬

The call to action button was modified by adjusting the font size for better readability and visual consistency. Its position was also changed to align centrally within its container div, while a hover effect was added to make it more interactive and visually appealing.
## Checklist ✅

- [ ] Reviewed the code changes thoroughly.
- [ ] Tested the changes locally and ensured they work as expected.
- [ ] Updated relevant documentation.
- [ ] Added appropriate test cases, if applicable.
- [ ] Considered potential security or performance implications.
- [ ] Code adheres to the project's style guidelines.
- [ ] Addressed all reviewer comments and concerns, if any.

## Screenshots (if applicable)

Screenshot (96)

## Additional Notes 📒

Include any additional information or context that might be helpful for the reviewers. 
***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,8 +9,6 @@
       "version": "0.1.0",
       "dependencies": {
         "@reduxjs/toolkit": "^1.9.3",
-        "@types/node": "18.15.11",
```
Comment (lalitmee) [inline package-lock.json:12]: remove this file from your changes as you have not installed any new package.

**Review by lalitmee — CHANGES_REQUESTED** [review]
Please fill out the PR description completely.

File: package-lock.json (hunk 1 above)
Comment (jibola2588) [reply package-lock.json]: okay, I will do just day

**Comment by jibola2588** [comment]
I have removed the file, and done the necessary corrections you have highlighted.  I'm in GSSOC'23, please help me add the GSSOC'23 label to my PR, and the issue with issue number #184

**Review by lalitmee — CHANGES_REQUESTED** [review]
Why are you deleting the `package-lock.json` file? Please revert it back

**Comment by PiyushKalyanpy** [comment]
Hello, We are closing this pull request as the changes we requested were not implemented.

**Pull request closed without merging**